### PR TITLE
Refactor invoke and remove deprecated commands

### LINKED ISSUE #422: Remove deprecated cli commands
## Describe the proposal

Remove all deprecated cli commands/flags for v1.0 release

### PR DESCRIPTION
**Pull request by yaron2**
Closes #422 and refactors the invoke command to add an optional `--verb` flag.

Removes `invokePost` and `invokeGet` commands.

### CODE REVIEW

**Review by mukundansundar — COMMENTED** [review]
Is it possible to add `DELETE` and `PUT` tests as well ?

**Comment by philliphoff** [comment]
@yaron2 We're not (yet) using the `dapr` CLI for method invocations in the extension; we're invoking the sidecar directly (because, at the time, the `dapr` CLI wouldn't recognize `daprd` instances it did not itself start.  So, this *shouldn't* break the extension.

**Comment by yaron2** [comment]
> @yaron2 We're not (yet) using the `dapr` CLI for method invocations in the extension; we're invoking the sidecar directly (because, at the time, the `dapr` CLI wouldn't recognize `daprd` instances it did not itself start. So, this _shouldn't_ break the extension.

Great, thanks for the clarification.

**Comment by yaron2** [comment]
> /cc @philliphoff this is a breaking change for the VS code extension.

Added.

File: pkg/standalone/invoke_test.go [hunk 1]
```diff
@@ -94,13 +95,55 @@ func TestInvoke(t *testing.T) {
...
+					Err: tc.listErr,
+				},
+			}
+			res, err := client.Invoke(tc.appID, tc.method, tc.resp, "DELETE")
 			if tc.errorExpected {
 				assert.Error(t, err, "expected an error")
 				assert.Equal(t, tc.errString, err.Error(), "expected error strings to match")
 			} else {
 				assert.NoError(t, err, "expected no error")
-				assert.Equal(t, tc.postResponse, res, "expected response to match")
```
Comment (mukundansundar) [inline pkg/standalone/invoke_test.go:103]: postResponse can be removed?

**Review by mukundansundar — APPROVED** [review]

File: pkg/standalone/invoke_test.go (hunk 1 above)
Comment (yaron2) [reply pkg/standalone/invoke_test.go:103]: removed

**Comment by yaron2** [comment]
This is approved by @msfussell

**Pull request merged**